rfcs/select-component.md

```diff
@@ -177,18 +177,8 @@ export default () => {
   * `SelectComponentIcon: ?React.ComponentType` component for all icons appearing in Select component. It's provided `$type: ICON.loop | ICON.clearTag | ICON.clearAll | ICON.selected` to setup corresponding icon of Select component
   * `Tag: ?React.ComponentType` component for selected options Tags shown in Input for multiple mode selection
   * `SelectSpinner: ?React.ComponentType` component for dropdown spinner if async options are loading
-* `onChange: func(e: SyntheticInputEvent, params: Object)`:
-  handler for events on trigger element when option are changing selection.  `params` has `type` indicating which action is performed, current `Option` with `id` and `label` of selected\unselected option and `selectedOptions` array of all of selected.
-* `onTextInputChange: func(e: SyntheticInputEvent)`:
-  handler for events on trigger element when text of search input (in Search mode has changed).
-* `onMouseEnter: func`:
-  handler for events on trigger element
-* `onMouseLeave: func`:
-  handler for events on trigger element
-* `onFocus: func`:
-  handler for events on trigger element
-* `onBlur: func`:
-  handler for events on trigger element
+* `onSelect: func({event: SyntheticInputEvent, type, option, selectedOptions})`:
```

Comment:
if we'd change from `onChange` to `onSelect`, we could reserve `onChange` for the input event - what do you think?

Reply:
Yes, like the idea.

Reply:
I remember `onTextInputChange` was renamed to it's current name due to ambiguous name of `onChange`, because `onChange` is used for main state change, whereas text input is a secondary change yet. So in case of select it's not obvious which change is meant
@schnerd @abmai

Reply:
Yep, `onChange` is a bit vague here I think. Any reason why we don't want to keep `onTextInputChange` or `onInputChange`?